### PR TITLE
Flip schema and catalog errors in MetadataUtil

### PR DESCRIPTION
**Pull request by jreslock**

### CODE REVIEW

File: presto-main/src/main/java/com/facebook/presto/metadata/MetadataUtil.java [hunk 1]
```diff
@@ -121,9 +121,9 @@ public static QualifiedTableName createQualifiedTableName(Session session, Node
         List<String> parts = Lists.reverse(name.getParts());
         String tableName = parts.get(0);
         String schemaName = (parts.size() > 1) ? parts.get(1) : session.getSchema().orElseThrow(() ->
-                new SemanticException(CATALOG_NOT_SPECIFIED, node, "Catalog must be specified when session catalog is not set"));
+                new SemanticException(SCHEMA_NOT_SPECIFIED, node, "Schema must be specified when session catalog is not set"));
```
Comment (electrum) [inline presto-main/src/main/java/com/facebook/presto/metadata/MetadataUtil.java]: The "when session catalog" part should also be updated to "when session schema"

File: presto-main/src/main/java/com/facebook/presto/metadata/MetadataUtil.java [hunk 2]
```diff
@@ -121,9 +121,9 @@ public static QualifiedTableName createQualifiedTableName(Session session, Node
...
         String catalogName = (parts.size() > 2) ? parts.get(2) : session.getCatalog().orElseThrow(() ->
-                new SemanticException(SCHEMA_NOT_SPECIFIED, node, "Schema must be specified when session schema is not set"));
+                new SemanticException(CATALOG_NOT_SPECIFIED, node, "Catalog must be specified when session schema is not set"));
```
Comment (electrum) [inline presto-main/src/main/java/com/facebook/presto/metadata/MetadataUtil.java]: Same as above

**Comment by electrum** [comment]
Thanks for finding this. Can you add some tests? I think we will need a new `TestMetadataUtil` class.

**Comment by martint** [comment]
Looks like this has already been fixed in master. Closing.

**Pull request closed without merging**